core/tests/behavior/utils.rs

```diff
@@ -110,255 +103,89 @@ macro_rules! blocking_trials {
     };
 }
 
-/// ObjectReaderFuzzer is the fuzzer for object readers.
-///
-/// We will generate random read/seek/next operations to operate on object
-/// reader to check if the output is expected.
-///
-/// # TODO
-///
-/// This fuzzer only generate valid operations.
-///
-/// In the future, we need to generate invalid operations to check if we
-/// handled correctly.
-pub struct ObjectReaderFuzzer {
-    name: String,
-    bs: Vec<u8>,
-
-    offset: usize,
-    size: usize,
-    cur: usize,
-    rng: ThreadRng,
-    actions: Vec<ObjectReaderAction>,
+pub struct Fixture {
+    pub paths: Mutex<Vec<String>>,
 }
 
-#[derive(Debug, Clone, Copy)]
-pub enum ObjectReaderAction {
-    Read(usize),
-    Seek(SeekFrom),
-    Next,
-}
-
-impl ObjectReaderFuzzer {
```

Comment:
Seems you removed those fuzz related tests? Did they be covered by other parts?

Reply:
Yep, we should run fuzz in a seperate target instead in behavior tests. The working loop will be:

- Running fuzzing target
- Extract fuzz cases into behavior tests
- Fix the this case